lib/modules/platform/github/index.md

```diff
@@ -51,9 +51,13 @@ The [GitHub App associated email](https://github.community/t/logging-into-git-as
 It needs to have the user id _and_ the username followed by the `users.noreply.`-domain of either github.com or the GitHub Enterprise Server.
 A way to get the user id of a GitHub app is to [query the user API](https://docs.github.com/en/rest/reference/users#get-a-user) at `api.github.com/user/self-hosted-renovate[bot]` (github.com) or `github.enterprise.com/api/v3/uer/self-hosted-renovate[bot]` (GitHub Enterprise Server).
 
-**`token:"x-access-token:${github-app-installation}"`**
+**`token:"ghs_123exampletoken"`**
 
-The token needs to be prefixed with `x-access-token` and be a [GitHub App Installation token](https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps#authenticating-as-an-installation).
+The token needs to be a [GitHub App Installation token](https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps#authenticating-as-an-installation).
+
+Previously, the token had to be prefixed with `x-access-token:`.
+While a prefixed token would still work, it's highly recommended to move away from it as its support will be removed in the future.
+Tokens starting with `ghs_` should not be prefixed, only tokens from GitHub Enterprise Server releases older than `3.2` need to be prefixed.
```

Comment:
This sentence is hard to understand. I think we should explain:

1. Who this sentence applies to
2. What versions this applies to
3. What you should do

How about something like this?

> If you're on a GitHub Enterprise Server _before_ version `3.2` then you must add a prefix to any token that starts with `ghs_`.

Reply:
> If you're on a GitHub Enterprise Server before version 3.2 then you must add a prefix to any token that starts with ghs_.

As far as I'm aware, tokens generated on GitHub Enterprise Server versions older than `3.2` do not start with `ghs_`. Tokens starting with `ghs_` do not need to be prefixed.
I would propose two alternatives:

> If you are on GitHub Enterprise Server before version 3.2 then you must add the `x-access-token:` prefix to the token.

or

> Any tokens that do not start with `ghs_` (for example tokens from GitHub Enterprise Server versions before version `3.2.`) must be prefixed with `x-access-token:`.

The second alternative is worded a little bit more general. What do you think @HonkingGoose ?

Reply:
Let's go with the second alternative. 😉